ee/hogai/session_summaries/session_group/summary_notebooks.py

```diff
@@ -190,27 +195,34 @@ def format_intermediate_state(self) -> TipTapNode:
         return {"type": "doc", "content": content}
 
 
-def _create_notebook_title(team_name: str) -> str:
-    return f"Session Summaries Report - {team_name} ({timezone.now().strftime('%Y-%m-%d')})"
+def _create_notebook_title(team_name: str, summary_name: str | None) -> str:
+    title = f"Session summaries report - {team_name}"
```

Comment:
We're thinking to add tags to notebooks, in this case you could tag this with "Session summaries" and avoiding adding this "Session summaries report" prefix to the title.